xmtp_mls/src/utils/test/tester_utils.rs

```diff
@@ -52,42 +63,67 @@ use xmtp_id::{
     },
     scw_verifier::SmartContractSignatureVerifier,
 };
-use xmtp_proto::api_client::ApiBuilder;
 use xmtp_proto::prelude::XmtpTestClient;
 use xmtp_proto::{TestApiBuilder, ToxicProxies};
+use xmtp_proto::{api_client::ApiBuilder, xmtp::message_contents::PrivateKey};
 
 type XmtpMlsProvider = XmtpOpenMlsProvider<Arc<TestMlsStorage>>;
 
 /// A test client wrapper that auto-exposes all of the usual component access boilerplate.
 /// Makes testing easier and less repetetive.
-pub struct Tester<Owner, Client>
+pub struct Tester<Owner = PrivateKeySigner, Client = FullXmtpClient>
 where
     Owner: InboxOwner,
 {
     pub builder: TesterBuilder<Owner>,
     pub client: Client,
     pub worker: Option<Arc<WorkerMetrics<SyncMetric>>>,
+    #[cfg(target_arch = "wasm32")]
     pub stream_handle: Option<Box<dyn StreamHandle<StreamOutput = Result<(), SubscribeError>>>>,
```

Comment:
if you use tokio_with_wasm, you can avoid this `StreamHandle` stuff, since the spawn apis are the same. at some point we need to remove `StreamHandle` entirely, since originally it was created to deal with differences between tokio & wasm_bindgen_futures

the [`xmtp_common::task::`](https://xmtp.github.io/libxmtp/xmtp_common/wasm/task/index.html) path should have the spawn re-exported. it has the advantage of using the nice tokio handle types as well. this will avoid the cfg trick here